msal-dotnet-articles/advanced/exceptions/wam-errors.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Errors associated with Web Account Manager (WAM)
+description: "When building applications using MSAL.NET and the Web Account Manager (WAM), developers may run into issues. This article explains potential errors and mitigations."
+ms.date: 08/30/2023
+---
+
+# Errors associated with Web Account Manager (WAM)
+
+Generally, errors generated by the MSAL component that interacts with the Web Account Manager (WAM) are [automatically converted](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/d25f45f7e356bd92986e412dd68b5d341343aa6a/src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs#L56) to an instance of <xref:Microsoft.Identity.Client.MsalException>. This enables developers to not worry about the internals and instead handle errors through idiomatic .NET constructs. However, for additional deep dives, developers will often need to investigate specific error messages.
+
+The table below highlights some of the most common errors along with potential mititgation strategies. For other exceptions, refer to the [Exceptions document](../../advanced/exceptions/index.md).
+
+>[!WARNING]
+>Error codes and error messages are presented for reference only. We do not recommend manually implementing exception handling strategies based on these and instead use the standard <xref:Microsoft.Identity.Client.MsalException>-based approach.
+
+| Error Code | Retry-able | Error message | Mitigation |
```

Comment:
I feel like `Retry-able` is a bit misleading here. Above we say to not retry on these error codes but on MsalException, but here it kind of implies to retry on these error codes. I would either remove the column or clarify above that this just says which type of MsalException to expect.

Reply:
+1, the Retry-able concept should only apply on the top layer, MSAL exceptions in this case, and there is already a mitigation so Retry-able seems not necessary here.